src/System.Net.Http/src/System/Net/Http/Headers/HttpRequestHeaders.cs

```diff
@@ -15,73 +15,49 @@ public sealed class HttpRequestHeaders : HttpHeaders
         private static readonly Dictionary<string, HttpHeaderParser> s_parserStore = CreateParserStore();
         private static readonly HashSet<string> s_invalidHeaders = CreateInvalidHeaders();
 
+        private const int AcceptSlot = 0;
+        private const int ExpectSlot = 1;
+        private const int IfMatchSlot = 2;
+        private const int IfNoneMatchSlot = 3;
+        private const int TransferEncodingSlot = 4;
+        private const int UserAgentSlot = 5;
+        private const int AcceptCharsetSlot = 6;
+        private const int AcceptEncodingSlot = 7;
+        private const int AcceptLanguageSlot = 8;
+        private const int NumCollectionsSlots = 9;
```

Comment:
nit:  why aren't these in alphabetical ordering as they are in numerical ordering?